scripts/upgrade-configs.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs-extra');
+const simpleGit = require('simple-git');
+const { fdir } = require("fdir");
+
+const major = process.env.npm_package_version.split('.')[0];
+
+async function replaceVersion(filePath){
+    const data = await fs.readFile(filePath, 'utf8');
+    const result = data.replace(/(@solid\/community-server\/)\^\d+\.\d+\.\d+/gm, `$1^${major}.0.0`);
+    return fs.writeFile(filePath, result, 'utf8');
+}
+
+async function upgradeConfig(){
+    console.log(`Changing @solid/community-server references to ${major}.0.0\n`)
+    
+    const configs = await new fdir().withBasePath()
```

Comment:
Since we simply need to find all json files in the config directory and below I would just make a small recursive function that does this.